tools/auction-deploy/tests/conftest.py

```diff
@@ -1,17 +1,30 @@
 import json
 import os
 
-import eth_tester
-import pytest
-from eth_keyfile import create_keyfile_json
-from eth_utils import to_canonical_address
-
-from auction_deploy.core import AuctionOptions
-
 # increase eth_tester's GAS_LIMIT
 # Otherwise we can't whitelist enough addresses for the validator auction in one transaction
-assert eth_tester.backends.pyevm.main.GENESIS_GAS_LIMIT < 8 * 10 ** 6
-eth_tester.backends.pyevm.main.GENESIS_GAS_LIMIT = 8 * 10 ** 6
+# This needs to be at the top of external imports when imported modules import eth_tester themselves
+import eth_tester  # noqa: E402, E702 isort:skip
+
+assert (
+    eth_tester.backends.pyevm.main.GENESIS_GAS_LIMIT < 8 * 10 ** 6
+)  # noqa: E402, E702 isort:skip
+eth_tester.backends.pyevm.main.GENESIS_GAS_LIMIT = (
+    8 * 10 ** 6
+)  # noqa: E402, E702 isort:skip
+
+
+import pytest  # noqa: E402, E702 isort:skip
```

Comment:
it is fine that the stuff above have the ignore linting stuff, but that here every line has it is a bit much. Please find a way to only have to exclude the eth_tester lines.

Reply:
Took me quite some time to figure it out. My understanding is that the flake8 ignore for `E402` has to be on every line that is an import below a non import statement or that the non import statement has to be on the same line as an import statement (for some reason).

Tell me if you like the current version better.